integration-cli/docker_cli_service_logs_test.go

```diff
@@ -277,7 +283,7 @@ func (s *DockerSwarmSuite) TestServiceLogsTTY(c *check.C) {
 	// and make sure we have all the log lines
 	waitAndAssert(c, defaultReconciliationTimeout, countLogLines(d, name), checker.Equals, 2)
 
-	cmd := d.Command("service", "logs", name)
+	cmd := d.Command("service", "logs", "--pretty=false", name)
```

Comment:
Can we reverse that? Having a boolean flag that requires a value to be useful is really awkward. The output In 17.05 is the "pretty" version? Or different?

Alternative `--raw`?